roaring.go

```diff
@@ -474,42 +474,67 @@ func (rb *Bitmap) Equals(o interface{}) bool {
 }
 
 // AddOffset adds the value 'offset' to each and every value in a bitmap, generating a new bitmap in the process
-func AddOffset(x *Bitmap, offset uint32) (answer *Bitmap) {
-	containerOffset := highbits(offset)
-	inOffset := lowbits(offset)
+func AddOffset(x *Bitmap, offset int64) (answer *Bitmap) {
```

Comment:
Unfortunately changing the function signature will break existing customers. Go doesn't do implicit type promotion. 

I believe the Go way would be to add a separate function.
> "Packages intended for public use should try to maintain backward compatibility as they evolve. The Go 1 compatibility guidelines are a good reference here: don't remove exported names, encourage tagged composite literals, and so on. If different functionality is required, add a new name instead of changing an old one. If a complete break is required, create a new package with a new import path."

https://github.com/golang/go/wiki/Modules#semantic-import-versioning

Reply:
@maciej You are definitely right! I completely forgot about existing clients :(. I will add a new method `AddOffsetInt64`.